uint8 MOVED=0
uint8 JAILED=1

ball_collector_robot/Target target

uint8 action
---
